ingestion_tools/scripts/common/finders.py

```diff
@@ -54,6 +56,8 @@ def __init__(
             name_regex = "(.*)"
         self.name_regex = re.compile(name_regex)
 
+        self.exclude_regexes = [re.compile(regex) for regex in exclude_regexes or []]
```

Comment:
It looks like we're supporting an `exclude_regexes` parameter for SourceGlobFinder as well as an `exclude` filter for any source type? It seems like the `exclude` filter would be enough and we don't need both?

Reply:
Good catch! I initially added `exclude_regexes` to `SourceGlobFinder`. But, decided to generalize it to exclude for all the source types. Looks like the `exclude_regexes` got missed in clean up.